packages/radix-ui-themes/src/components/base-checkbox.css

```diff
@@ -28,6 +28,11 @@
   position: absolute;
   width: var(--checkbox-indicator-size);
   height: var(--checkbox-indicator-size);
+
+  /* Required for subpixel alignment in Safari */
+  transform: translate(-50%, -50%);
+  top: 50%;
+  left: 50%;
```

Comment:
Current situation:

![Screenshot 2024-03-26 at 12 59 38](https://github.com/radix-ui/themes/assets/8441036/b5b7bb75-7afd-4d2c-aaba-4fe40ad5d6fe)

Reply:
hmm weird… we didn't add a new size in 3.0 right?

Reply:
Nope